llvm/test/Transforms/DeadStoreElimination/stores-of-existing-values.ll

```diff
@@ -531,7 +533,9 @@ define void @test12_memset_other_store_in_between_partial_overlap(ptr %ptr) {
 
 define void @test12_memset_later_store_exceeds_memset(ptr %ptr) {
 ; CHECK-LABEL: @test12_memset_later_store_exceeds_memset(
-; CHECK-NEXT:    call void @llvm.memset.p0.i64(ptr align 1 [[PTR:%.*]], i8 0, i64 8, i1 false)
+; CHECK-NEXT:    [[TMP1:%.*]] = getelementptr inbounds i8, ptr [[PTR:%.*]], i64 5
+; CHECK-NEXT:    call void @llvm.memset.p0.i64(ptr align 1 dereferenceable(5) [[TMP1]], i8 0, i64 5, i1 false)
```

Comment:
This looks wrong? We used to have a single memset writing 8 bytes, and now we got two memset writing 5+4=9 bytes.

Reply:
Sorry for the delay, fixed in https://github.com/llvm/llvm-project/pull/75478/commits/bc32bcb6d8e7c10a8a141370540fda3adf932d30.